src/ServiceControl.Config/UI/SharedInstanceEditor/SharedServiceControlEditorViewModelValidator.cs

```diff
@@ -42,7 +42,7 @@ protected List<string> UsedQueueNames(TransportInfo transportInfo = null, string
                     p.ErrorQueue,
                     p.AuditQueue,
                     p.AuditLogQueue
-                })
+                }).Where(queuename => string.Compare(queuename, "!disable", true) != 0)
```

Comment:
From MSDN 

![image](https://user-images.githubusercontent.com/174258/38424044-18273332-39b0-11e8-80dc-64e3395264e9.png)

https://msdn.microsoft.com/en-us/library/zkcaxw5y(v=vs.110).aspx

Reply:
Nice! I did not know that